### PR TITLE
Navigation Structure Refactor

### PR DESCRIPTION
**Pull request by MHHukiewitz**
This is a big one, so I'm just going to link to the discussion in the forum:

https://community.aleph.im/t/aleph-docs-refactoring/147

A couple of new pages have been added and should be considered drafts (generated with ChatGPT, need more links, factual checks, etc):

- Roadmap 2024 (based on Medium article)
- Page linking to Medium blog with all partnership announcements
- How to use aleph.im with $ALEPH requirements
- Potential use cases and all Aleph capabilities listed for dev inspiration

### CODE REVIEW

**Comment by MHHukiewitz** [comment]
Okay, so this is how far I'm going with the refactoring. I added a couple of pages, mainly kept the file structure as-is, which will prevent links from changing.

There's still a couple of topics that need to be taken care of, like the reverse oracle and instances. I will add Issues for them.

**Comment by MHHukiewitz** [comment]
Okay, so this is how far I'm going with the refactoring. I added a couple of pages, mainly kept the file structure as-is, which will prevent links from changing.

There's still a couple of topics that need to be taken care of, like the reverse oracle and instances. I will add Issues for them.

File: docs/tools/vrf.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+We provide a script to deploy the VM functions.
+Just run the following command to package the application and upload it to the aleph.im network.
+
+```
+python3 deployment/deploy_vrf_vms.py
+```
+
+If the deployment succeeds, the script will display links to the VMs on the aleph.im network. Example:
+
+> Executor VM: https://api2.aleph.im/api/v0/messages/558b0eeea54d80d2504b0287d047e0b78458d08022d3600bcf8478700dd0aac2
```
Comment (Antonyjin) [inline docs/tools/vrf.md:46]: The quote is too long so it overtakes on the right navigation
image


Could it be better if we just put this as a block using: **```** ?

so it looks like this:
image

File: mkdocs.yml
Comment (Antonyjin) [inline mkdocs.yml:1]: image

Seems that the pages: 

- computing/runtimes/existing.md
- tools/aleph-tooling.md

were removed but not deleted inside the repo.

Should be deleted.

**Review by Antonyjin — APPROVED** [review]
Apart from some very minor changes to make, this new structure is very good. I find it clearer with different sections about the community and development, and within these sections, specific information is well provided.

As for the new documentation page, it's well explained with good examples, it is very good.

**Pull request merged**